Text/index.jsx

```diff
@@ -20,7 +21,7 @@ Text.propTypes = {
   children: PropTypes.node,
   size: PropTypes.oneOf(['large', 'small', 'extra-small']),
   weight: PropTypes.oneOf(['bold', 'thin']),
-  color: PropTypes.oneOf(['white']),
+  color: PropTypes.oneOf(['white', 'gray']),
```

Comment:
I love that idea, @hharnisc. Might we be thinking of reducing our color palette further? 😄 I'd love to explore how we can make our color ratio more a11y, too!